Gruntfile.js

```diff
@@ -98,17 +70,11 @@ module.exports = function(grunt) {
     },
     'webpack-dev-server': {
       options: {
-        webpack: webpackConfig,
-        publicPath: '/build/'
-      },
-      dev: {
-        hot: true,
-        port: 3000,
-        https: getDevCerts() || true,
-        webpack: {
-          devtool: 'source-map'
+        output: {
+          publicPath: '/build/'
         }
       },
+      dev: webpackConfig,
```

Comment:
All this webpack dev server config (along with `getDevCerts()` was moved into `webpack.config.js`.